src/CustomMarshalers/src/System/Runtime/InteropServices/CustomMarshalers/EnumerableViewOfDispatch.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.InteropServices.ComTypes;
+using System.Text;
+
+namespace System.Runtime.InteropServices.CustomMarshalers
+{
+    internal class EnumerableViewOfDispatch : ICustomAdapter, IEnumerable
+    {
+        private const int DispId_NewEnum = -4;
```

Comment:
nit: PascalCasing for constants (remove underscore)

Also, why is it -4?

Reply:
It's -4 because it's a reserved DISPID. See https://msdn.microsoft.com/en-us/library/cc237818.aspx. It's the standardized DISPID for getting an enumerator. That way any `IDispatch`-implementing COM interface can provide an enumerator easily.

Reply:
Can you please add a comment in source as well explaining that (and anywhere else we have seemingly magic constants that have special meaning).